api-specifications/admin-api/admin-api-2.2.0.yaml

```diff
@@ -0,0 +1,2383 @@
+openapi: 3.0.1
+info:
+  version: 2.2.0
+  description: The Ed-Fi Admin API is an API-based administrative interface for the Ed-Fi ODS / API Platform.  It provides functions for vendor, application and other aspects of ODS / API management, it also support the Data Management Service.
```

Comment:
This is still too specific to our implementations (ODS/API and Data Management Service). The reason for publishing a specification is so that other people can use it for supporting their own custom Ed-FI API applications.